src/Provider/Nominatim/Tests/NominatimTest.php

```diff
@@ -109,6 +110,17 @@ public function testGeocodeWithRealAddress()
         $this->assertEquals('yes', $result->getType());
     }
 
+    public function testGeocodeWithRealAddressThatReturnsOptionalQuarter()
+    {
+        $provider = Nominatim::withOpenStreetMapServer($this->getHttpClient(), 'Geocoder PHP/Nominatim Provider/Nominatim Test');
+        $results = $provider->geocodeQuery(GeocodeQuery::create('woronicza 17, warszawa, polska'));
+
+        $this->assertCount(1, $results);
+
+        /* @var NominatimAddress $result */
```

Comment:
```suggestion
        /* @var \Geocoder\Provider\Nominatim\Model\NominatimAddress $result */
```

For coherence!
You can then remove line 18.

Reply:
:+1: